### PR TITLE
added FAH_SARS-CoV-2_Mpro_FEP.yml

### PR DESCRIPTION
**Pull request by yabmtm**
## Description
Provide a brief description of the data being added here.

## Status
- [ ] YAML file for each piece of data
- [ ] Ready to go

### CODE REVIEW

File: data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+type: md
+title: Trajectories of the SARS-CoV-2 main protease protein from Folding@Home
+description: >
+  All-atom MD simulations of the SARS-CoV-2 main protease with small molecules, simulated using Folding@Home.
```
Comment (jchodera) [inline data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml:4]: "with small molecules from the COVID Moonshot"?

Don't forget you can include links, just as if this were markdown!

File: data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  Each small molecule was run alone in solution and docked to Mpro to compute absolute free energies of binding.
```
Comment (jchodera) [inline data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml:5]: Link to where the small molecules came from?

File: data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  Simulations were run using GROMACS-5.0.4 or GROMACS-2020 and are stored as compressed binary XTC files.
```
Comment (jchodera) [inline data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml:6]: PDB files defining the topology in the XTC files are critical to these. Can you be sure to show how to retrieve those before you show how to retrieve the XTCs? Also, do they include solute only, solute and ions, all atoms?

File: data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  Absolute free energies were computed using an expanded ensemble scheme, using 40 lambdas to decouple the ligand interactions.
```
Comment (jchodera) [inline data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml:7]: Links to methodology, if possible? And to the input files?

File: data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  Free energy data for each alchemical lambda is stored in the md.log file, and found more concisely in pre-scraped pkl dataframes.
+  The dataset comprises several projects, each having a RUN*/CLONE*/result* directory structure.
```
Comment (jchodera) [inline data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml:9]: You can use single backticks to protect `RUN*/CLONE*/result*` and render it as code.

File: data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  Each PROJ represents a different dataset of small molecules. Each RUN represents a different small molecule,
+  each CLONE is a unique MD run differing in initial atomic velocities, and result is a fragment of the contiguous simulation.
+  For the case of ligand-only "L" simulations, each xtc represents 10ns of sampling, while each complex "RL" xtc trajectory is only 1ns.
+  More information about the simulation set-up can be found [here](https://github.com/FoldingAtHome/covid-moonshot/blob/master/folding-at-home_submissions/COVID%20Moonshot%20051220%20Release%20Announcement.pdf).
+  In order to find particular PROJ/RUN of interest, see the results and organization dataframes.
+  
+  *Entire dataset:* To get the raw trajectory files (XTCs) of the whole dataset (~10+ TB/project), you can use the [AWS CLI](https://aws.amazon.com/cli/):
```
Comment (jchodera) [inline data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml:16]: Can you lead with how to get distilled subsets, or a single trajectory for each ligand, or retrieve data for a single ligand? Do you have a master table of ligands somewhere?

File: data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+   ```
+   *Build directories:* To pull the build directory for a given project (including structure, topology, MD parameters for a given project/run, access via [AWS CLI](https://aws.amazon.com/cli):
+    ```bash
+    aws s3 --no-sign-request sync s3://fah-public-data-covid19-absolute-free-energy/build/p14721 .
+    aws s3 --no-sign-request sync s3://fah-public-data-covid19-absolute-free-energy/build/p14722/RUN0 .
+    aws s3 --no-sign-request sync s3://fah-public-data-covid19-absolute-free-energy/build/p14725/RUN10/npt.gro .
+    ```
+  *Organization dataframe:* In order to determine systems of interest, you can parse [this dataframe](https://fah-public-data-covid19-absolute-free-energy/free_energy_data/organization.pkl) which contains information on which ligands are present in each project/run. Half of the projects contain trajectories for ligands alone in solution, while the other half contain trajectories for ligands docked to the main protease.  More data is being added to this repository every day, so if a project/run is not available in the AWS bucket, check back later.
```
Comment (jchodera) [inline data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml:31]: This might go first, then the build directories, then results, then the raw datasets.

**Review by jchodera — COMMENTED** [review]
Looking good! Just some comments on how to improve this!

File: data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml (hunk 6 above)
Comment (yabmtm) [reply data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml:16]: My master table of ligands is the organization.pkl, which shows identity, project, run information for all protease simulations. I am leading with organization, then topology file, where I show examples for downloading build directories for an entire project, build directory for a particular run, solvated structure file, and structure file with just solute. The raw datasets section similarly shows examples for grabbing varying amounts of trajectory data.

File: data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml (hunk 4 above)
Comment (yabmtm) [reply data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml:7]: I have a link to methodology that we wrote up for our Moonshot release. Do you think this is sufficient? The input files are also accounted for and I'm transferring them to the bucket today.

Comment (jchodera) [reply data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml:8]: Yes, thanks!

**Review by jchodera — APPROVED** [review]
Looks good!

**Comment by jchodera** [comment]
@Lnaden Can you help us with the schema validation errors?

**Comment by yabmtm** [comment]
@Lnaden I fixed the schema validation issue. Should be ready to merge now.

**Pull request merged**